.circleci/config.yml

```diff
@@ -66,15 +66,7 @@ jobs:
       - setup_integration_test
       - run:
           name: Test
-          command: bash -i -c 'npx cypress run --record --config numTestsKeptInMemory=1 --spec cypress/integration/group3/**/*'
-  integration_test_4:
-    executor: integration_test_exec
-    working_directory: ~/repo
-    steps:
-      - setup_integration_test
-      - run:
-          name: Test
-          command: bash -i -c 'npx cypress run --record --config numTestsKeptInMemory=1 --spec cypress/integration/group4/**/*'
+          command: bash -i -c 'npx cypress run --record --config numTestsKeptInMemory=1 --spec cypress/integration/testThatDoNotModifyDatabase/**/*'
```

Comment:
How does the database get initialized for these tests? Clearly it works, since the tests passed, but I'm just wondering. resetDB() would not only reset the database after a test ran so the next test started with a clean slate, but presumably for the very first test it would set the state.

Reply:
Wouldn't it just be going off the state it's left in after the last test in group 2?

Reply:
I think they run in parallel, not sequentially. Plus the tests might need the "pristine" state, rather than the state left over from other tests.

I'm pretty sure this works, just not getting how. :)

Reply:
Ohh right. Is it the second to last run command in this file then?

Reply:
Ah... I don't think so, but that gave me a hint; it's a few above:

```      - run:
          name: Prepare webservice
          command: bash -i -c 'npm run webservice'
```
That runs a script which does it.

Thanks.

@garyluu I'm good on this.